pkg/asset/manifests/vsphere/cloudproviderconfig.go

```diff
@@ -27,7 +27,7 @@ func CloudProviderConfig(clusterName string, p *vspheretypes.Platform) (string,
 	printIfNotEmpty(buf, "server", p.VCenter)
 	printIfNotEmpty(buf, "datacenter", p.Datacenter)
 	printIfNotEmpty(buf, "default-datastore", p.DefaultDatastore)
-	printIfNotEmpty(buf, "folder", clusterName)
+	printIfNotEmpty(buf, "folder", p.Folder)
```

Comment:
@dav1x can you make one change?
if `p.Folder` is empty use `clusterName`

Reply:
Yes. done!